packages/block-library/src/navigation/edit/index.js

```diff
@@ -82,6 +85,9 @@ function Navigation( {
 	hasColorSettings = true,
 	customPlaceholder: CustomPlaceholder = null,
 } ) {
+	const isOffCanvasNavigationEditorEnabled =
+		window?.__experimentalEnableOffCanvasNavigationEditor === true;
```

Comment:
Is it OK to access window here? Just wondering if this could be accessed from contexts where `window` is not defined (like mobile apps).

Reply:
Yes, that is how we handle experiments. There is no other way I've seen to look into serverside set flags. Not sure what react native does about window, but the condition is `window?` (notice the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)) so if it doesn't exist `isOffCanvasNavigationEditorEnabled` is just `null`